object-manipulation-3/main.js

```diff
@@ -0,0 +1,269 @@
+  },
+  {
+    name: 'P2',
+    hand: []
+  },
+  {
+    name: 'P3',
+    hand: []
+  },
+  {
+    name: 'P4',
+    hand: []
+  }
+];
+const deck = [
+  {
+    rank: 'ace',
+    suit: 'spades'
+  },
+  {
+    rank: '2',
+    suit: 'spades'
+  },
+  {
+    rank: '3',
+    suit: 'spades'
+  },
+  {
+    rank: '4',
+    suit: 'spades'
+  },
+  {
+    rank: '5',
+    suit: 'spades'
+  },
+  {
+    rank: '6',
+    suit: 'spades'
+  },
+  {
+    rank: '7',
+    suit: 'spades'
+  },
+  {
+    rank: '8',
+    suit: 'spades'
+  },
+  {
+    rank: '9',
+    suit: 'spades'
+  },
+  {
+    rank: '10',
+    suit: 'spades'
+  },
+  {
+    rank: 'jack',
+    suit: 'spades'
+  },
+  {
+    rank: 'queen',
+    suit: 'spades'
+  },
+  {
+    rank: 'king',
+    suit: 'spades'
+  },
+  {
+    rank: 'ace',
+    suit: 'hearts'
+  },
+  {
+    rank: '2',
+    suit: 'hearts'
+  },
+  {
+    rank: '3',
+    suit: 'hearts'
+  },
+  {
+    rank: '4',
+    suit: 'hearts'
+  },
+  {
+    rank: '5',
+    suit: 'hearts'
+  },
+  {
+    rank: '6',
+    suit: 'hearts'
+  },
+  {
+    rank: '7',
+    suit: 'hearts'
+  },
+  {
+    rank: '8',
+    suit: 'hearts'
+  },
+  {
+    rank: '9',
+    suit: 'hearts'
+  },
+  {
+    rank: '10',
+    suit: 'hearts'
+  },
+  {
+    rank: 'jack',
+    suit: 'hearts'
+  },
+  {
+    rank: 'queen',
+    suit: 'hearts'
+  },
+  {
+    rank: 'king',
+    suit: 'hearts'
+  },
+  {
+    rank: 'ace',
+    suit: 'clubs'
+  },
+  {
+    rank: '2',
+    suit: 'clubs'
+  },
+  {
+    rank: '3',
+    suit: 'clubs'
+  },
+  {
+    rank: '4',
+    suit: 'clubs'
+  },
+  {
+    rank: '5',
+    suit: 'clubs'
+  },
+  {
+    rank: '6',
+    suit: 'clubs'
+  },
+  {
+    rank: '7',
+    suit: 'clubs'
+  },
+  {
+    rank: '8',
+    suit: 'clubs'
+  },
+  {
+    rank: '9',
+    suit: 'clubs'
+  },
+  {
+    rank: '10',
+    suit: 'clubs'
+  },
+  {
+    rank: 'jack',
+    suit: 'clubs'
+  },
+  {
+    rank: 'queen',
+    suit: 'clubs'
+  },
+  {
+    rank: 'king',
+    suit: 'clubs'
+  },
+  {
+    rank: 'ace',
+    suit: 'diamonds'
+  },
+  {
+    rank: '2',
+    suit: 'diamonds'
+  },
+  {
+    rank: '3',
+    suit: 'diamonds'
+  },
+  {
+    rank: '4',
+    suit: 'diamonds'
+  },
+  {
+    rank: '5',
+    suit: 'diamonds'
+  },
+  {
+    rank: '6',
+    suit: 'diamonds'
+  },
+  {
+    rank: '7',
+    suit: 'diamonds'
+  },
+  {
+    rank: '8',
+    suit: 'diamonds'
+  },
+  {
+    rank: '9',
+    suit: 'diamonds'
+  },
+  {
+    rank: '10',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'jack',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'queen',
+    suit: 'diamonds'
+  },
+  {
+    rank: 'king',
+    suit: 'diamonds'
+  }
+];
+*/
+
+//   function game(players, hand) {
+//  const shuff = _.shuffle(deck);
+//  const deal = _.chunk(shuff, [2]);
+
+//  players.push(deal)[1];
+//  console.log(players);
+//  shuffle the deck. w/ _.shuffle();
+//  deal 2 cards to each player. how idk. w/ _.chunk().
+// push? the deck into 'hand' arrays... for...in loop?
```

Comment:
to deal cards to the player, you can take the shuffled array of cards and either use other lodash array methods like `_first` or a native javascript array method like `splice`